cloud/scope/machine.go

```diff
@@ -155,6 +155,19 @@ func (m *MachineScope) CreateMachine() (*vpcv1.Instance, error) {
 			}
 			instancePrototype.Keys = append(instancePrototype.Keys, key)
 		}
+	} else if m.IBMVPCMachine.Spec.SSHKeysRef != nil {
+		instancePrototype.Keys = []vpcv1.KeyIdentityIntf{}
+		for _, sshKey := range m.IBMVPCMachine.Spec.SSHKeysRef {
+			keyID, err := fetchKeyID(sshKey, m)
+			if err != nil {
+				record.Warnf(m.IBMVPCMachine, "FailedRetrieveKey", "Failed key retrieval - %v", err)
```

Comment:
should be error here right? which key? can this be more specific to ssh key?

Reply:
Fixed.